tests/TestCase.php

```diff
@@ -100,11 +105,12 @@ protected function getPackageProviders($app)
         ];
     }
 
+ 
     /**
      * @param  \Illuminate\Foundation\Application  $app
      * @return void
      */
-    protected function getEnvironmentSetUp($app)
+    protected function getEnvironmentSetup($app) 
```

Comment:
Don't see why this had to change. The method is called exactly how it is in the `orchestra/testbench` package